src/dom/Draggable.js

```diff
@@ -75,7 +75,7 @@ export var Draggable = Evented.extend({
 
 		// If we're currently dragging this draggable,
 		// disabling it counts as first ending the drag.
-		if (L.Draggable._dragging === this) {
+		if (this._dragging === this) {

```

Comment:
This should just be `_dragging`, since it's actually referring to a class global ("static" in Java/C# terminology) defined here: https://github.com/atd-schubert/Leaflet/blob/21f31fa825acaaab0f78663c00854c45b99db16b/src/dom/Draggable.js#L24